NuKeeper/Commands/CollaborationPlatformCommand.cs

```diff
@@ -132,21 +132,34 @@ protected override async Task<int> Run(SettingsContainer settings)
         {
             var fileSettings = FileSettingsCache.GetSettings();
 
-            if (!Platform.HasValue)
+
+            bool defaultValue;
+
+            // The default value is true, if it is supported for the corresponding platform.
+            if (Platform.HasValue &&
+                Platform != Abstractions.CollaborationPlatform.Platform.AzureDevOps
+                && Platform != Abstractions.CollaborationPlatform.Platform.GitLab
+                && Platform != Abstractions.CollaborationPlatform.Platform.Bitbucket)
```

Comment:
Is this the right way around? Bitbucket (I believe) supports delete after merge, bitbucket local doesn't support it. Might be good to move this list to be shared with lines 158-158 too? Thanks :)

Reply:
Bitbucket/BitbucketLocal:
This is true, Bitbucket does support it and BitbucketLocal doesn't. 

Right way around:
Yes, it is the right way around as this condition is true when there is a platform specified and this platform is NOT one of those tthree. Thus, the default value is going to be false in case of an unsupported platform.

Shared List:
Yes, I'll do it.

Reply:
Btw. a general question&remark which arised while working on #798 & #843
It is currently hard to work with platform related things as it is not possible to see what platform is used, unless it was specified explicitly. This is also the case there. It is still possible that the wrong default value is used, if a platform was chosen from the URL.
I don't have a good overview over the code base but from what I saw, it would make sense to not tie the selection of the platform to the Settings reader but rather do it before the settings reader is chosen.
Concrete, that means, evaluating the platform first (by Url, passed argument or file) and set the correct enum value. Then it doesn't matter anymore from where this value originated.